src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,64 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  movies: Movie[];
+  onAdd: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ movies, onAdd }) => {
+  const [query, setQuery] = useState('');
+  const [isLoading, setIsloadig] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [movie, setMovie] = useState<Movie | null>(null);
+
+  const clear = () => {
+    setQuery('');
+    setIsError(false);
+    setMovie(null);
+  };
+
+  const handleOnAdd = (newMovie: Movie) => {
+    if (movies.every(prevMovie => prevMovie.imdbId !== newMovie.imdbId)) {
```

Comment:
you can move it check in the onAdd function, and in that case it's no need to pass movies list in the FindMovie component